src/manipulateToDo.js

```diff
@@ -45,20 +82,153 @@ const drop = (ev) => {
+      elUnder.style.color = '#fff';
+      Array.from(elUnder.childNodes).forEach((node) => {
+        if (node.nodeName === 'DIV') {
+          const nodes = node.childNodes;
+          Array.from(nodes).forEach((one) => {
+            one.style.backgroundColor = '#4c3c3c';
+            one.style.color = '#fff';
+          });
+        }
+        node.style.backgroundColor = '#4c3c3c';
+        node.style.color = '#fff';
+      });
       setTimeout(() => {
-        document.getElementById(id).style.backgroundColor = '#fff';
+        // document.getElementById(id).style.backgroundColor = "#fff";
+        let elUnder = document.getElementById(id);
+        // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+        const classArray = elUnder.classList.value.split(' ');
+        if (
+          classArray.includes('fa-check')
+          || classArray.includes('checkbox')
+          || classArray.includes('to-do')
+        ) {
+          elUnder = elUnder.parentElement.parentElement;
+        } else if (
+          classArray.includes('fa-arrows-alt')
+          || classArray.includes('two')
+        ) {
+          elUnder = elUnder.parentElement;
+        }
+        elUnder.style.backgroundColor = '#fff';
+        elUnder.style.color = '#fff';
+        Array.from(elUnder.childNodes).forEach((node) => {
+          if (node.nodeName === 'DIV') {
+            const nodes = node.childNodes;
+            Array.from(nodes).forEach((one) => {
+              one.style.backgroundColor = '#fff';
+              one.style.color = 'rgb(134, 122, 122)';
+            });
+            // if (task.completed) {
+            //   Array.from(Array.from(elUnder.childNodes)[0].childNodes)[2].style.color = '#ccc';
+            // } else {
+
+            // }
+          }
+          node.style.backgroundColor = '#fff';
+          node.style.color = 'rgb(204, 187, 187)';
+          if (node.classList.value.includes('fa-arrows-alt')) {
+            node.addEventListener('mouseover', () => {
+              node.style.color = 'rgb(75, 66, 66)';
+            });
+            node.addEventListener('mouseout', () => {
+              node.style.color = 'rgb(204, 187, 187)';
+            });
+          }
+        });
       }, 810);
     }, 10);
   } else if (prevNextIds.prevId > prevNextIds.nextId) {
     setTimeout(() => {
-      document.getElementById(
-        id,
-      ).style.backgroundColor = '#4c3c3c';
+      let elUnder = document.getElementById(id);
+      // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+      const classArray = elUnder.classList.value.split(' ');
+      if (
+        classArray.includes('fa-check')
+        || classArray.includes('checkbox')
+        || classArray.includes('to-do')
+      ) {
+        elUnder = elUnder.parentElement.parentElement;
+      } else if (
+        classArray.includes('fa-arrows-alt')
+        || classArray.includes('two')
+      ) {
+        elUnder = elUnder.parentElement;
+      }
```

Comment:
This same code is repeating through this file several times. Making it a separate function would improve readability and stick a bit better to the YAGNI, what do you think? 🤓